src/coreclr/scripts/superpmi_replay.py

```diff
@@ -23,13 +23,17 @@
 parser.add_argument("-platform", help="OS platform")
 parser.add_argument("-jit_directory", help="path to the directory containing clrjit binaries")
 parser.add_argument("-log_directory", help="path to the directory containing superpmi log files")
+parser.add_argument("-partition", help="Partition number specifying which set of flags to use: 1 or 2")
 
-jit_flags = [
+jit_flags1 = [
```

Comment:
Sure, that's fine and general. I went for simplicity. We'd want to pass to pass both "partition#" and "total partition count" from the proj file, then any change would only need to be made in the proj file.

Reply:
Perhaps, the better approach would be to partition based on collection type and that way it would create (existing partitions X collection count). With that, we will download one type of collection for each platform/architecture and run *all* jitstressregs on that partition. You might have to figure out how to magically write all of that in `.proj` file.

```cmd
    <SPMI_Partition Include="win-x64" Platform="windows" Architecture="x64" CollectionFilter="benchmark"/>    
    <SPMI_Partition Include="win-x64" Platform="windows" Architecture="x64" CollectionFilter="libraries.pmi"/>
    <SPMI_Partition Include="win-x64" Platform="windows" Architecture="x64" CollectionFilter="crossgen2"/>
   ...

    ; likewise for win-arm64, unix-x64, linux-arm64, osx-arm64
```

Reply:
I like that because of the download efficiency. However, the different collections are very different sizes, so some partitions will be much slower (e.g., libraries_tests). Also, it's not great to hard-code the set of collections we have, since currently we'll download any and all -- especially true for benchmark & aspnet which currently are not all-platform.